pages/index.vue

```diff
@@ -11,7 +11,15 @@
           <h3>Watch</h3>
         </button>
       </div>
-      <div v-html="video" class="player" />
+      <div v-if="isPlaying" class="player">
+        <iframe
+          src="https://player.vimeo.com/video/576805134?color=ff0086&title=0&byline=0&portrait=0&autoplay=true"
+          class="video-iframe"
+          frameborder="0"
+          allow="autoplay; fullscreen"
+          allowfullscreen
+        ></iframe>
```

Comment:
Very minor thing but can this be a self closing tag?

Reply:
Hey @DaisyyWard are iframe valid html if their self closed?